wp-content/plugins/hurumap/src/FlourishBlock/Save.js

```diff
@@ -8,6 +8,13 @@ function Save({ attributes }) {
       data-chart-id={attributes.chartId}
       data-chart-title={attributes.title}
       data-chart-description={attributes.description}
+      data-show-insight={attributes.showInsight}
+      data-insight-title={attributes.insightTitle}
+      data-insight-summary={attributes.insightSummary}
+      data-data-link-title={attributes.dataLinkTitle}
+      data-analysis-country={attributes.analysisCountry}
+      data-analysis-link-title={attributes.analysisLinkTitle}
+      data-data-geo-id={attributes.dataGeoId}
```

Comment:
Same as above, lets define props to match what we've defined in the HURUmap UI

Also, if possible, lets group these props. i.e. all analysis props should be in one group, then all data props, etc.